File: responses/5U6H43-FGP/count_of_largest_integer.py
# you can write to stdout for debugging purposes, e.g.
# print("this is a debug message")

import re

def solution(S):
    array = re.findall('[-+]\d+', S)
    
    _sum = 0
    for each in array:
        each = int(each)
        _sum += each   
        
    return _sum

File: responses/MKQGAM-AWS/count_of_largest_integer.js
// you can write to stdout for debugging purposes, e.g.
// console.log('this is a debug message');

function solution(S) {
    var sum=0;
    for(var i=0;i<S.length;i++){
        if(S[i]=='-'){
            sum-=parseInt(S[i+1]);
        }
        else if(S[i]=='+'){
            sum+=parseInt(S[i+1]);
        }
    }
    return sum;
}